wrappers/vroom.rb

```diff
@@ -42,6 +42,7 @@ def solver_constraints
         :assert_no_subtours,
         :assert_points_same_definition,
         :assert_single_dimension,
+        :assert_no_planning_heuristic,
```

Comment:
This assert is relative to the resolution method, it should belong to the Solver catgory